src/configs.rs

```diff
@@ -31,6 +39,13 @@ lazy_static! {
         AtomicI64::new(BLOOM_MEMORY_LIMIT_PER_FILTER_DEFAULT);
     pub static ref BLOOM_USE_RANDOM_SEED: AtomicBool = AtomicBool::default();
     pub static ref BLOOM_DEFRAG: AtomicBool = AtomicBool::new(BLOOM_DEFRAG_DEAFULT);
+    pub static ref BLOOM_FP_RATE_DEFAULT_STRING: String = BLOOM_FP_RATE_DEFAULT.to_string();
+    pub static ref BLOOM_FP_RATE: ValkeyGILGuard<ValkeyString> = ValkeyGILGuard::new(
+        ValkeyString::create(None, BLOOM_FP_RATE_DEFAULT.to_string())
+    );
+    pub static ref TIGHTENING_RATIO_DEFAULT_STRING: String = TIGHTENING_RATIO.to_string();
```

Comment:
Can you add an actual f64 Mutex above this? We need to add this and change the f64 mutex.

```suggestion
    pub static ref BLOOM_TIGHTENING_F64: Mutex<f64> = Mutex::new(TIGHTENING_RATIO);
    pub static ref BLOOM_TIGHTENING_DEFAULT: String = TIGHTENING_RATIO.to_string();
```